openy.packages.yml

```diff
@@ -168,3 +193,89 @@ locations:
+  - openy_popups
+
+camps:
+  name: 'Camps Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+    - openy_prgf_camp_menu
+    - openy_loc_camp
+
+programs:
+  name: 'Programs Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+    - openy_prgf_repeat_schedules
+    - openy_prgf_repeat_schedules_loc
+    - openy_prgf_categories_listing
+    - openy_prgf_classes_listing
+    - openy_prgf_class_location
+    - openy_prgf_class_sessions
+    - openy_prgf_schedule_search
+    - openy_node_activity
+    - openy_node_category
+    - openy_node_class
+    - openy_node_session
+    - openy_node_program
+
+groupexpro:
+  name: 'Integration: GroupEx Pro Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+    - openy_prgf_embedded_groupexpro_schedule
+    - openy_prgf_group_schedules
+
+activenet:
+  name: 'Integration: ActiveNet Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+  - activenet
+
+daxko:
+  name: 'Integration: Daxko Package'
+  help: '<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+  <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor
+  quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean
+  ultricies mi vitae est. Mauris placerat eleifend leo.</p>'
+  description: '...'
+  modules:
+  - daxko
```

Comment:
there is also another module daxko2, I think it should be there as well

Reply:
Added that